packages/manager/modules/product-offers/src/workflows/product-offers-order-workflow.class.js

```diff
@@ -24,7 +24,8 @@ export default class OrderWorkflow extends Workflow {
    *  - {serviceNameToAddProduct}: Service name of which we will add product/addon.
    *  If set, the order will consist to add option to an existing service.
    *  If null, the order concerns an new product.
-   *  - {planCode}: planCode to use. Plan code example : webHosting, cloudDB.
+   *  - {getPlanCode}:
```

Comment:
Maybe we could add explanation on why this needs to be a function and cannot be just a string

Reply:
Done !

Reply:
Done !